core/main/src/main.rs

```diff
@@ -30,7 +30,7 @@ pub mod state;
 pub mod utils;
 include!(concat!(env!("OUT_DIR"), "/version.rs"));
 
-#[tokio::main]
+#[tokio::main(worker_threads = 2)]
```

Comment:
Is there any specific reason for configuring worker_threads count to 2 ?

Reply:
Based on analysis with the new changes 2 worker threads seem more than enough for handling the load of Ripple. Logic behind this is we have 2 websockets one for 3474 and 3473. So the worker threads will be used by tokio runtime to sequester the next tokio task in one of the workers depending on the availability. 2 would be more optimal than over allocation of 4